pulp_rpm/app/modulemd.py

```diff
@@ -211,3 +212,34 @@ def parse_modular(file):
             logging.warning(f"Unknown modular document type found: {parsed_data.get('document')}")
 
     return modulemd_all, modulemd_defaults_all, modulemd_obsoletes_all
+
+
+class ModularYamlLoader(yaml.SafeLoader):
+    """Custom Loader that preserve unquoted float in specific fields"""
```

Comment:
Can we leave a bit more detail, including the issue number where this was originally encountered, and perhaps point to the libmodulemd equivalent code

Reply:
Also, where this code in particular came from

Reply:
Sure